exercises/exercise_6/README.md

```diff
@@ -1 +1,519 @@
+ => [internal] load .dockerignore                                                                                                                                               0.0s
+ => => transferring context: 2B                                                                                                                                                 0.0s
+ => [internal] load build definition from Dockerfile                                                                                                                            0.0s
+ => => transferring dockerfile: 109B                                                                                                                                            0.0s
+ => [internal] load metadata for docker.io/library/alpine:3.9.3                                                                                                                 0.3s
+ => CACHED [1/2] FROM docker.io/library/alpine:3.9.3@sha256:28ef97b8686a0b5399129e9b763d5b7e5ff03576aa5580d6f4182a49c5fe1913                                                    0.0s
+ => [internal] load build context                                                                                                                                               0.1s
+ => => transferring context: 86B                                                                                                                                                0.0s
+ => [2/2] COPY cnab/app/run /cnab/app/run                                                                                                                                       0.1s
+ => exporting to image                                                                                                                                                          0.0s
+ => => exporting layers                                                                                                                                                         0.0s
+ => => writing image sha256:03addb3d4bab4e50084921db9a14decf1bc7a86fe18d62c7e7676c33665e8e5c                                                                                    0.0s
+ => => naming to docker.io/library/simple:1.0.0-invoc                                                                                                                           0.0s
+```
+
+We have now a `bundle.json` AND an invocation image `simple:1.0.0-invoc`, congratulations you have made your first CNAB bundle :tada:
+It's time to play with it using Docker App!
+
+## Docker App CNAB Runtime
+
+### Use Docker App as a standard CNAB runtime
+
+Docker App is a standard CNAB runtime. It can execute the following actions on any CNAB bundle:
+- install
+- upgrade
+- io.cnab.status
+- uninstall
+
+**EXERCISE**: Let's play a little with our bundle, try to install, status, upgrade with another port, status again, then uninstall.
+
+<details>
+    <summary>Solution</summary>
+
+```sh
```

Comment:
May want to break this up to a few smaller code blocks, as I'm sure folks will miss commands and/or get lost in output vs commands they should run.